cogs/memes.py

```diff
@@ -151,17 +151,22 @@ def __init__(self, bot):
         await ctx.message.delete()
 
     @commands.command()
-    async def boot(self, ctx, num: int = 2):
-        """Draws a pyramid of boots, default is 3 unless user specifies
+    async def boot(self, ctx, num: int = 2, emoji: str = "👢"):
+        """Draws a pyramid of boots (or any other emoji), default is 2 unless user specifies
```

Comment:
2 things here:

Could you break the line at 80 chars? The help output looks kind of strange otherwise.

![image](https://user-images.githubusercontent.com/12765385/71550060-eb229700-2995-11ea-86a4-39132e041b78.png)

Additionally, could you include in the docstring that the function accepts any string (and any multiword string in quotes) as an argument instead of an emoji?